source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
```

Comment:
This is only relevant if you're trying to run static locally too. You shouldn't need to do this if you just want to run a frontend app (e.g: government-frontend) pointing at live static.